tailwind.config.js

```diff
@@ -15,22 +15,19 @@ module.exports = {
         "bd-yellow-dark": "#F7CB47",
       },
     },
-    maxWidth: {
```

Comment:
This is blocking all TW default max-w-classes. I think we only need the defaults in the future.

Reply:
Will need to check the dashboard anyway before we merge the setup-redesign branch 👍